### PR TITLE
Finalize overall design for Homepage

### PR DESCRIPTION
**Pull request by jittaearn**
I have edited the design of the home page both before and after user authentication. I have also added a new page for intruction where players can learn how to play our game. Please review the overall design. If there's any suggestion please leave a comment below.

### CODE REVIEW

**Comment by MaquiaSA** [comment]
Both forms.py and models.py miss empty blank line at the end of file

File: game/forms.py [hunk 1]
```diff
@@ -1,7 +1,40 @@
...
+        )
+    )
+    hint = forms.CharField(label=mark_safe('Hint</br>'), max_length=100000,
+    widget=forms.Textarea(
+        attrs={
+            'id': 'hint',
+            'style' : 'height: 50px; font-size: 22px'
+            }
+        )
+    )
```
Comment (jittaearn) [inline game/forms.py]: Add blank line after code for better style.

**Comment by jittaearn** [comment]
@MaquiaSA Thank you for your suggestion. I have fixed the code styling for form.py. Please recheck the changes.

**Comment by magmagcup** [comment]
Approve, all code following code script

**Comment by Champ2k** [comment]
Don't show allowed host in our projects, please hide with decouple.

**Comment by jittaearn** [comment]
@Champ2k I've hidden the Allowed_Hosts  to the env. file using Django. Please review my work.

**Comment by MaquiaSA** [comment]
Approve

**Comment by Champ2k** [comment]
Good, Approved

**Comment by jittaearn** [comment]
@MaquiaSA @magmagcup @Champ2k  I have fixed the conflict between branch template and develop. Please try if it works fine on your laptop.

**Comment by Champ2k** [comment]
Screen Shot 2562-11-30 at 11 29 56

Statistic page can't be found.

**Comment by magmagcup** [comment]
Statistic page work now, merge approve.

**Comment by MaquiaSA** [comment]
Everything works well. Merge approved.

**Comment by Champ2k** [comment]
The statistic page is now working, merge approved.

**Pull request merged**